testonly/tmock/fake_server.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package fake
+package tmock
```

Comment:
Is there a godoc package description for this somewhere which needs updating too?

Reply:
`Package tmock is a generated GoMock package.`  Would you recommend putting the fake server in it's own package?